src/containers/views/Configuration.js

```diff
@@ -43,7 +44,7 @@ export class Configuration extends Component {
         <Editor
           editorChanged={editorChanged}
           onEditorChange={onEditorChange}
-          json={config}
+          content={toYAML(config)}
```

Comment:
[reference](https://github.com/jekyll/jekyll-admin/issues/246#issuecomment-268996338) to another issue